appendix/solutions.md

```diff
@@ -1308,12 +1308,14 @@ corresponding flag will eventually spoiler the language code `tlh_AA`.
    JSON to `jwtn3d@juice-sh.op`.
 4. Change the value of the `alg` property in the `header` part from
    `HS256` to `none`.
-5. Re-encode the JWT and then delete the signature part so that the
-   final character of the JWT is the last `.` (dot symbol).
-6. Change the `Authorization` header of a subsequent request to the
+5. Encode the `header` to `base64url`. Similarly, encode the `payload`
+   to `base64url`.
+6. Join the two strings obtained above with a `.` (dot symbol) and
+   add a `.` at the end of the obtained string. So, effectively it
+   becomes `base64url(header).base64url(payload).`
+7. Change the `Authorization` header of a subsequent request to the
    retrieved JWT (prefixed with `Bearer ` as before) and submit the
-   request. Alternatively you can set the `token` cookie to the JWT
```

Comment:
Why did you remove that sentence? Did this not work for you? This had nothing to do with jwt.io, it's a fake legacy behavior in the Juice Shop.

Reply:
No, it works. I actually didn't understand it in one go. So, I was unsure if it makes sense with the changes I am making. Now I understand what it actually meant. I am really sorry for not seeing it carefully. I will go ahead with add it again with the PR.

<img alt="screenshot 2019-02-21 at 6 51 14 am" width="1680" src="https://user-images.githubusercontent.com/35000671/53136491-34030580-35a5-11e9-8682-0d715cc85494.png">

Reply:
Hi @bkimminich I feel the test is failing independent of my change( build history shows it's failing for every commit). Can you please confirm?